hphp/hack/src/server/autocompleteService.ml

```diff
@@ -143,9 +143,9 @@ let autocomplete_token ac_type x =
 
 let autocomplete_id id = autocomplete_token Acid id
 
-let get_class_elt_types env class_ cid elts =
+let get_class_elt_types ~is_method env class_ cid elts =
   let is_visible (_, elt) =
-    Tast_env.is_visible env (elt.ce_visibility, get_ce_lsb elt) cid class_
+    Tast_env.is_visible ~is_method env (elt.ce_visibility, get_ce_lsb elt) cid class_
```

Comment:
Wondering if there's a better way to do this... given a value of type `class_elt`, can we retrieve the fact that it references a method or property? I wasn't sure how, which is why I'm threading that information down from here.